lib/plugins/aws/lib/naming.js

```diff
@@ -485,4 +485,16 @@ module.exports = {
   getCustomResourceCognitoUserPoolResourceLogicalId(functionName, idx) {
     return `${this.getNormalizedFunctionName(functionName)}CustomCognitoUserPool${idx}`;
   },
+  // Event Bridge
+  getCustomResourceEventBridgeHandlerFunctionName() {
+    return 'custom-resource-event-bridge';
```

Comment:
does this not cause issues if you try to use the custom resource in multiple projects?

Reply:
🤔 it shouldn't since the handler is only defined once per service.

Reply:
oh. handler function name. i was thinking lambda name.